_posts/2019-02-12-native-elm-datepicker.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "Native Elm Datepicker"
+date: 2019-04-12 5:00:00 -0500
+category: blog
+author: RJ
+author_twitter:
+image:
```

Comment:
Why prefer imageless on this one?

Reply:
Should rename this field `twitter_image`